.circleci/config.yml

```diff
@@ -21,6 +22,7 @@ jobs:
           paths:
             - /usr/local/cargo/registry
             - /usr/local/cargo/git
+            - /usr/local/cargo/bin/cargo-junit
```

Comment:
Just reading the documentation of the `cargo-junit` crate: https://github.com/CWood1/cargo-junit#usage

and just like every cargo extension you need to use `cargo` to launch your extension once the extension installed.

i.e.:

```
cargo install cargo-junit
cargo junit --name test_output.xml
```